src/Header/RequestHandler.h

```diff
@@ -19,12 +19,19 @@
 class RequestHandler
 {
 public:
-    //instantiate instance with request, will parse to OperateData and check with Authenticator and call correspond api.
-    RequestHandler(const std::string &requestData);
+    //instantiate instance with accepted file descriptor, will receive request string and parse to OperateData, then check with Authenticator and call correspond api.
+    RequestHandler(const int &file_descriptor);
```

Comment:
why use file descriptor?

Reply:
As mentor said that handler should keep waiting for same user request, or else the MainManager will stock by the user who is connecting but not sending request with multithread.

Reply:
understood.

Reply:
This is reasonable.